release.config.cjs

```diff
@@ -0,0 +1,6 @@
+/**
+ * @type {import('semantic-release').GlobalConfig}
+ */
+module.exports = {
+  branches: ['release']
```

Comment:
Seems simpler to put this in the package.json file perhaps?
```json
{
  "release": {
    "branches": ["release"]
  }
}
```

Reply:
Good point, done